index.js

```diff
@@ -9,13 +9,13 @@ var restoreErrorAndWarnings = spy.restoreErrorAndWarnings;
 
 beforeEach(watchErrorAndWarnings);
 
-afterEach(function(done) {
-	var actual = filterErrorAndWarnings(/(Invalid prop|Failed prop type|Unknown prop)/);
+afterEach(function (done) {
+	var actual = filterErrorAndWarnings(/(Invalid prop|Failed prop type|Unknown prop|non-boolean attribute|Received NaN|Invalid value|React does not recognize|React uses onFocus and onBlur instead of onFocusIn and onFocusOut|Invalid event handler property|Unknown event handler property|Directly setting property `innerHTML` is not permitted|The `aria` attribute is reserved for future use in |for a string attribute `is`. If this is expected, cast|Invalid DOM property)/);
```

Comment:
I removed the duplicate entires.  Should the checks for 'aria', 'is' and 'innerHtml' be changed to wildcards to catch other properties that may fall into the same category?  Are those the only examples?